packages/tools/src/tools/base/ContourBaseTool.ts

```diff
@@ -30,6 +29,30 @@ abstract class ContourBaseTool extends AnnotationTool {
     super(toolProps, defaultToolProps);
   }
 
+  static getContourSequence(toolData, metadataProvider) {
+    const { data } = toolData;
+
+    const ContourData = [];
+    for (const point of data.contour.polyline) {
+      for (const v of point) {
+        ContourData.push(v.toFixed(2));
+      }
+    }
+
+    const { referencedImageId } = toolData.metadata;
+    const ContourImageSequence = metadataProvider.get(
+      'ImageSopInstanceReference',
+      referencedImageId
+    );
+
+    return {
+      NumberOfContourPoints: ContourData.length / 3,
+      ContourImageSequence,
+      ContourGeometricType: 'CLOSED_PLANAR',
```

Comment:
Why are we forcing it to be closed planar when we know a contour in `ContourBaseTool` can be open? Maybe we should rename the function to indicate it's specifically for `closed_planar` contours.